core/esp_spi.c

```diff
@@ -23,8 +23,8 @@
 #define _SPI1_SCK_GPIO  14
 #define _SPI1_CS0_GPIO  15
 
-#define _SPI0_FUNC 1
-#define _SPI1_FUNC 2
+#define _SPI0_FUNC IOMUX_FUNC(1)
+#define _SPI1_FUNC IOMUX_FUNC(2)
```

Comment:
As I suggested in https://github.com/SuperHouse/esp-open-rtos/pull/171 the function `gpio_set_iomux_function` might be added in *gpio.h*. In that case `_set_pin_function` will not be needed here. Also I'm not sure if _SPIx_FUNC defines are necessary here as they already defined in iomux_regs.h